core/bin/zksync_api/src/bin/dev_ticker_server.rs

```diff
@@ -0,0 +1,42 @@
+use actix_cors::Cors;
+use actix_web::{middleware, App, HttpServer};
+use providers::{
+    dev_liquidity_provider::config_liquidity_app, dev_price_provider::create_price_service,
+};
+use structopt::StructOpt;
+
+mod providers;
+
+#[derive(Debug, StructOpt, Clone, Copy)]
+struct FeeTickerOpts {
+    /// Activate "sloppy" mode.
+    ///
+    /// With the option, server will provide a random delay for requests
+    /// (60% of 0.1 delay, 30% of 0.1 - 1.0 delay, 10% of 5 seconds delay),
+    /// and will randomly return errors for 5% of requests.
+    #[structopt(long)]
+    sloppy: bool,
+}
+
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
+    let _vlog_guard = vlog::init();
+
+    let opts = FeeTickerOpts::from_args();
+    if opts.sloppy {
+        vlog::info!("Fee ticker server will run in a sloppy mode.");
+    }
+
+    HttpServer::new(move || {
+        App::new()
+            .wrap(Cors::default().send_wildcard().max_age(3600))
+            .wrap(middleware::Logger::default())
+            .configure(config_liquidity_app)
+            .service(create_price_service(opts.sloppy))
+    })
+    .bind("0.0.0.0:9876")
```

Comment:
could we update this, so it can configurable?

Reply:
well that would certainly make sense, however:
1. this is unchanged from the orig impl
2. we might not care what port it is on internally (in docker), instead we probably just want to be able to configure the port binding upon container creation
3. we'd have to add the config to
  1. dev_ticker config
  2. fee_ticker config (coingecko_url)
  4. Dockerfile